study-builder/studies/osteo/study.conf

```diff
@@ -1815,6 +1830,20 @@
       "dispatchToHousekeeping": true,
       "order": 1
     },
+    {
+      "trigger": {
+        "type": "INVITATION_EMAIL"
```

Comment:
Sorry for not being clear, what we should do here is actually:

* Use trigger type `INVITATION_CREATED`
* Use action type `INVITATION_EMAIL`

This way, after the invitation is created by the event above, we will then send out an email with the invitation guid embedded.